### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by utkarshani**

### CODE REVIEW

**Comment by utkarshani** [comment]
created causalPattern folder

**Comment by utkarshani** [comment]
pull

**Comment by utkarshani** [comment]
pull

File: causalPattern/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Causal pattern ODP
+https://w3id.org/causalPattern -> https://github.com/utkarshani/CausalPattern
+
+Contact
+Utkarshani Jaimini - utkarshani2009@gmail.com
+Cory henson - coryhenson@gmail.com
```
Comment (TallTed) [inline causalPattern/README.md:6]:
```suggestion
# Causal pattern ODP

https://w3id.org/causalPattern -> https://github.com/utkarshani/CausalPattern

## Contacts
* Utkarshani Jaimini - utkarshani2009@gmail.com
* Cory henson - coryhenson@gmail.com
```

**Pull request merged**